### PR TITLE
Tree-shake Change

### PR DESCRIPTION
**Pull request by schmidt-sebastian**

### CODE REVIEW

**Review by hsubox76 — APPROVED** [review]
LG, just one question

File: packages/database/src/core/view/Change.ts [hunk 1]
```diff
@@ -32,44 +47,33 @@ export class Change {
     public oldSnap?: Node,
     public prevName?: string | null
   ) {}
+}
```
[the remark this answers is not in the document]
Comment (schmidt-sebastian) [reply packages/database/src/core/view/Change.ts:46]: We can make it an interface, which completely removes this type from the final bundle.

File: packages/database/src/core/view/Change.ts [hunk 2]
```diff
@@ -17,59 +17,68 @@
...
 
-  static childChangedChange(
-    childKey: string,
-    newSnapshot: Node,
-    oldSnapshot: Node
-  ): Change {
-    return new Change(Change.CHILD_CHANGED, newSnapshot, childKey, oldSnapshot);
-  }
+/** Event type for a value change */
+export const CHANGE_TYPE_VALUE = 'value';
```
Comment (Feiyang1) [inline packages/database/src/core/view/Change.ts:33]: Can these `const` variables be `const enum`? Can shave a little bit of size.

Comment (Feiyang1) [reply packages/database/src/core/view/Change.ts:33]: Also will be a bit easier to use.

Comment (schmidt-sebastian) [reply packages/database/src/core/view/Change.ts]: Done. Thanks for the suggestion

Comment (schmidt-sebastian) [reply packages/database/src/core/view/Change.ts]: -500 bytes by the way

**Pull request merged**